servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcStatus.java

```diff
@@ -144,8 +171,11 @@ public GrpcStatusException asException() {
      *
      * @param applicationStatusSupplier the {@link Supplier} for the {@link Status}.
      * @return the current status wrapped in a {@link GrpcStatusException}.
+     * @deprecated Use {@link GrpcStatusException#GrpcStatusException(GrpcStatus)}.
```

Comment:
`GrpcStatusException#of(Status)` seems like a better alternative for `GrpcStatus#asException(Supplier<Status>)`